bentoml/_internal/frameworks/catboost.py

```diff
@@ -114,51 +108,45 @@ def load(
         booster = bentoml.catboost.load("my_model:latest", model_params=dict(model_type="classifier"))
     """  # noqa
 
-    _, _model_file, _model_params = _get_model_info(tag, model_params, model_store)
+    _, _model_file, _model_params = _get_model_info(tag, model_params)
 
     return _load_helper(_model_file, _model_params)
 
 
-@inject
 def save(
     name: str,
     model: "CatBoostModel",
     *,
     model_params: t.Optional[t.Dict[str, t.Union[str, t.Any]]] = None,
     model_export_parameters: t.Optional[t.Dict[str, t.Any]] = None,
-    model_pool: t.Optional["cbt.core.Pool"] = None,
     labels: t.Optional[t.Dict[str, str]] = None,
     custom_objects: t.Optional[t.Dict[str, t.Any]] = None,
     metadata: t.Union[None, t.Dict[str, t.Any]] = None,
-    model_store: "ModelStore" = Provide[BentoMLContainer.model_store],
 ) -> Tag:
     """
     Save a model instance to BentoML modelstore.
 
     Args:
-        name (:code:`str`):
-            Name for given model instance. This should pass Python identifier check.
-        model (:code:`Union[catboost.core.CatBoost, catboost.core.CatBoostClassifier, catboost.CatBoostRegressor]`):
-            Instance of model to be saved
-        model_params (:code:`Dict[str, Union[str, Any]]`, `optional`, default to :code:`None`):
-            Parameters for a CatBoost model. Following parameters can be specified:
-                - model_type(:code:`str`): :obj:`classifier` (`CatBoostClassifier`) or :obj:`regressor` (`CatBoostRegressor`)
-        model_export_parameters (:code:`Dict[str, Union[str, Any]]`, `optional`, default to :code:`None`):
-            Export parameters for given model.
-        model_pool (:code:`cbt.core.Pool`, `optional`, default to :code:`None`):
-            CatBoost data pool for given model.
-        labels (:code:`Dict[str, str]`, `optional`, default to :code:`None`):
-            user-defined labels for managing models, e.g. team=nlp, stage=dev
-        custom_objects (:code:`Dict[str, Any]]`, `optional`, default to :code:`None`):
-            user-defined additional python objects to be saved alongside the model,
+        name (``str``):
+            The name for given model instance. This must be a valid :obj:`~bentoml._internal.types.Tag` name; see
```

Comment:
I think there's the links to doc of `Tag` after "see"?

Reply:
Going to just remove the 'see' part until we have real `Tag` docs.